meta-zarhus-webkit/recipes-networking/cog/cog/cog-autostart.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Service for autostarting Cog on boot.
+After=nginx.service weston.service
+
+[Service]
+Restart=always
+RestartSec=2
+Type=simple
+Environment=XDG_SESSION_TYPE=tty WAYLAND_DISPLAY=/run/wayland-0 XDG_SESSION_CLASS=user XDG_SESSION_ID=c2 XDG_RUNTIME_DIR=/run/user/ COG_PLATFORM_WL_VIEW_WIDTH="1920" COG_PLATFORM_WL_VIEW_HEIGHT="1080"
```

Comment:
What if width/height is different? You should probably detect with/height automatically e.g.

```sh
$ grep . /sys/class/drm/*/modes
/sys/class/drm/card1-eDP-1/modes:1920x1080
```

Or allow those variables to be set in a recipe. You could create file with variables to be set and use
[EnvironmentFile](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#EnvironmentFile=) and [PassEnvironment](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#PassEnvironment=)

Reply:
Maybe `COG_PLATFORM_FDO_VIEW_FULLSCREEN=1 cog` would work without setting size explicitly?

Reply:
Added configuration with `sed` here 21ba3ea2c2e03a8891edb3cdd363fac7aa7228f6.